crates/rocksdb/src/lib.rs

```diff
@@ -510,8 +510,12 @@ impl RocksDb {
         self.manager.async_spawn(task).await?
     }
 
+    pub async fn close(&self) -> Result<(), RocksError> {
```

Comment:
I'd have expected self to be moved here.

Reply:
I wanted to do that - the trouble is that in `RocksDbStorage` we hold on to an `Arc<RocksDb>` which I wasn't sure how to deal with?